java/bench/spark/src/java/org/apache/orc/bench/spark/SparkBenchmark.java

```diff
@@ -74,7 +74,8 @@
 @BenchmarkMode(Mode.AverageTime)
 @OutputTimeUnit(TimeUnit.MICROSECONDS)
 @AutoService(OrcBenchmark.class)
-@Fork(jvmArgsAppend = "--add-opens=java.base/sun.nio.ch=ALL-UNNAMED")
+@Fork(jvmArgsAppend = {"--add-opens=java.base/sun.nio.ch=ALL-UNNAMED",
+    "--add-opens=java.base/sun.util.calendar=ALL-UNNAMED"})
```

Comment:
Could you make a PR for these two lines, @cxzl25 ?

Reply:
Sure.